src/java.base/share/classes/java/text/Collator.java

```diff
@@ -138,7 +138,7 @@ public abstract class Collator
      * Collator strength value.  When set, only SECONDARY and above differences are
      * considered significant during comparison. The assignment of strengths
      * to language features is locale dependent. A common example is for
-     * different accented forms of the same base letter ("a" vs "\u00E4") to be
+     * different accented forms of the same base letter ("a" vs "ä") to be
```

Comment:
Since this (and the other one in RuleBasedCollator) is in the explanation of text handling, I think keeping the original code point makes sense. So I'd have both UTF-8 string and its Unicode escape notation here.

Reply:
Yes, I meant literally `\u00e9` or `\u00E4`, but I think it is better described as `U+00E9` emphasizing the code point. So in this case, I'd suggest
```
"a" vs "ä" (U+00E9)
```